### PR TITLE
Update 'Infrastructure List' documentation

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Update the documentation following https://github.com/DataDog/datadog-agent/pull/17857 which enabled this feature by default. This change targets agent 7.47 and should not be merged before this version of the agent is release.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/infrastructure/list.md [hunk 1]
```diff
@@ -85,17 +85,18 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
+
+  Starting with Agent version >= 7.47/6.47, this feature is enabled by default.
 {{< /callout >}}
 
 The Agent can send its own configuration to Datadog to be displayed in the `Agent Configuration` section of the host detail panel.
 
 The Agent configuration is scrubbed of any sensitive information and only contains configuration you've set using the configuration file or environment variables. The configuration changes are updated every 10 minutes.
 
-This feature is disabled by default. To enable it, add the following settings to your [Agent configuration file][6]:
+This feature is enabled by default since Agent version 7.47.0/6.47.0.
```
Comment (sgnn7) [inline content/en/infrastructure/list.md:96]:
```suggestion
This feature is enabled by default in Agent version >= 7.47.0/6.47.0.
```

File: content/en/infrastructure/list.md [hunk 2]
```diff
@@ -85,17 +85,18 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
 
-```yaml
-inventories_configuration_enabled: true
-```
+To enable or disable it, change the value of `inventories_configuration_enabled` setting in your [Agent configuration file][6] (set it to `true` to
```
Comment (sgnn7) [inline content/en/infrastructure/list.md:98]:
```suggestion
To modify this behavior, change the value of `inventories_configuration_enabled` setting in your [Agent configuration file][6] (set it to `true` to
```

File: content/en/infrastructure/list.md [hunk 3]
```diff
@@ -85,17 +85,18 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
+enable it and `false` to disable it):
```
Comment (sgnn7) [inline content/en/infrastructure/list.md:99]:
```suggestion
enable send of the configuration and `false` to disable it):
```

File: content/en/infrastructure/list.md [hunk 4]
```diff
@@ -85,17 +85,18 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
 
 Alternatively, use the `DD_INVENTORIES_CONFIGURATION_ENABLED=true` environment variable to enable this feature.
```
Comment (sgnn7) [inline content/en/infrastructure/list.md:101]:
```suggestion
Alternatively, use the `DD_INVENTORIES_CONFIGURATION_ENABLED` environment variable to enable or disable this feature.
```

**Review by sgnn7 — APPROVED** [review]
Left just a couple of tiny nits but LGTM

File: content/en/infrastructure/list.md [hunk 5]
```diff
@@ -85,19 +85,19 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
 
 The Agent configuration is scrubbed of any sensitive information and only contains configuration you've set using the configuration file or environment variables. The configuration changes are updated every 10 minutes.
 
-This feature is disabled by default. To enable it, add the following settings to your [Agent configuration file][6]:
+This feature is enabled by default in Agent version >= 7.47.0/6.47.0.
 
-```yaml
-inventories_configuration_enabled: true
-```
+To modify this behavior, change the value of `inventories_configuration_enabled` setting in your [Agent configuration file][6] (true to send the configuration and false to disable it).
```
Comment (rtrieu) [inline content/en/infrastructure/list.md:98]:
```suggestion
To modify this behavior, set the value of the `inventories_configuration_enabled` setting in your [Agent configuration file][6] to `true` to send the configuration, or `false` to disable it.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor suggestion, but otherwise looks good!

File: content/en/infrastructure/list.md [hunk 6]
```diff
@@ -85,19 +85,19 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
 
 The Agent configuration is scrubbed of any sensitive information and only contains configuration you've set using the configuration file or environment variables. The configuration changes are updated every 10 minutes.
 
-This feature is disabled by default. To enable it, add the following settings to your [Agent configuration file][6]:
+This feature is enabled by default in Agent version >= 7.47.0/6.47.0.
 
-```yaml
-inventories_configuration_enabled: true
-```
+To modify this behavior, set the value of the `inventories_configuration_enabled` setting in your [Agent configuration file][6] to `true` to send the configuration, or `false` to disable it.
```
Comment (sgnn7) [inline content/en/infrastructure/list.md:98]: @hush-hush Tiny nit:
```suggestion
To modify this behavior, set the value of the `inventories_configuration_enabled` in your [Agent configuration file][6] to `true` to send the configuration, or `false` to disable it.
```

File: content/en/infrastructure/list.md [hunk 7]
```diff
@@ -85,19 +85,19 @@ Datadog creates aliases for host names when there are multiple uniquely identifi
...
 
 The Agent configuration is scrubbed of any sensitive information and only contains configuration you've set using the configuration file or environment variables. The configuration changes are updated every 10 minutes.
 
-This feature is disabled by default. To enable it, add the following settings to your [Agent configuration file][6]:
+This feature is enabled by default in Agent version >= 7.47.0/6.47.0.
 
-```yaml
-inventories_configuration_enabled: true
-```
+To modify this behavior, set the value of the `inventories_configuration_enabled` in your [Agent configuration file][6] to `true` to send the configuration, or `false` to disable it.
```
Comment (rtrieu) [inline content/en/infrastructure/list.md:98]:
```suggestion
To modify this behavior, set the value of `inventories_configuration_enabled` in your [Agent configuration file][6] to `true` to send the configuration, or `false` to disable it.
```

**Review by rtrieu — APPROVED** [review]
Approving with a minor suggestion!

**Review by urseberry — APPROVED** [review]

**Pull request merged**